app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+
+
+
+
+
+
+<div>
+<section >
+  <h1>
+    ¿Qué tengo que hacer?
+  </h1>
+  <ul>
+    <il>
+      <%= link_to "Añadir Tarea", new_task_path %>
+    </il>
+    <il>
+      <%= link_to "Todas las tareas", tasks_path %>
```

Comment:
This section of links to "Añadir Tarea" and "Todas las tareas" is repeated at the top of every view. We can DRY up this code by moving it into the `app/views/layouts/application.html.erb` _layout template_.

In this case we would put it above the line with `<%= yield %>` so that it appears above all of the content for a given page.